thirdparty/ipc/ipc.h

```diff
@@ -0,0 +1,75 @@
+#ifndef AF_UNIX_IPC_INCLUDE
+#define AF_UNIX_IPC_INCLUDE
+
+#include <errno.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#ifdef _WIN32
+#include <winsock2.h>
+#include <ws2tcpip.h>
+#include <mswsock.h>
+#include <afunix.h>
+#else
+#include <sys/socket.h>
+#include <sys/un.h>
+#endif // _WIN32
+
+/* Inter process communication system 
+ * - Platform agnostic using AF_UNIX sockets
+ *   On Mac we must use DGRAM mode.
+ *   We'd like to make this connectionless would be easier code wise.
+ */
+
+using CallbackDefinition = void (*)(const char * /* string data received */, int /*strlen */);
+#define BufferSize 256
+
+// TODO: this should be dynamic
```

Comment:
Will you resolve this todo? If not, please document the limitation.

Reply:
yes this was done in a later update will be updating the changes here tomorrow

Reply:
fixed